boards/arduino-uno/Makefile.include

```diff
@@ -0,0 +1,46 @@
+# define the cpu used by the arduino uno board
+export CPU = atmega328p
+
+export TERMPROG = $(RIOTBASE)/dist/tools/pyterm/pyterm
+export TERMFLAGS = -b 9600 -p $(PORT)
+
+#define the flash-tool and default port depending on the host operating system
+OS = $(shell uname)
+ifeq ($(OS),Linux)
+  PORT ?= /dev/ttyACM0
+  FLASHER = avrdude
+else ifeq ($(OS),Darwin)
+  PORT ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+  FLASHER = avrdude
+else
+  $(info CAUTION: No flash tool for your host system found!)
+  # TODO: fix for building under windows
+endif
+export FLASHER
+export PORT
+export DIST_PATH = $(RIOTBOARD)/$(BOARD)/dist
+export DEBUGSERVER_PORT = 4242
+export DEBUGSERVER = $(DIST_PATH)/debug_srv.sh
+export DEBUGSERVER_FLAGS = "-g -j usb :$(DEBUGSERVER_PORT)"
+export DEBUGGER_FLAGS = "-x $(RIOTBOARD)/$(BOARD)/dist/gdb.conf $(ELFFILE)"
+export DEBUGGER = $(DIST_PATH)/debug.sh $(DEBUGSERVER_FLAGS) $(DIST_PATH) $(DEBUGSERVER_PORT)
+
+# PROGRAMMER defaults to stk500v2 which is the internal flasher via USB. Can be
+# overridden for debugging (which requires changes that require to use an ISP)
+export PROGRAMMER ?= stk500v2
```

Comment:
I have an arduino uno and this programmer doesn't work for me. Replacing `stk500v2` with `arduino` here and in the line below works btw.